pages/Messages/Serialization.md

```diff
@@ -13,7 +13,7 @@ menu: Messages
 | 0.1     | Tentative |
 
 ## Purpose
-1. Describe the form and method of DSNP message serialization.
+1. Describe the form and method of DSNP message serialization for posting messages in Ethereum logs.
```

Comment:
Is the Eth logs or the Batch Entries? Or perhaps both?

Reply:
It means, as opposed to serializing messages in response to queries to Batchers or Archivists.